packages/storefront/src/pages/component-v3-status.md

```diff
@@ -57,7 +57,7 @@
 | ⛔ **Tabs** / **Tabs Bar**  |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
 | ✅ **Pagination**           | - Text scale 200% is almost perfect because of the negative calc and max px clamp definition of fluid button size<br />- Add a test with new grid util ensuring pagination works on 320px viewport width<br />- get rid of counterReset<br />- we shouldn't loose focus by tabindex="0" when prev/next buttons are getting aria-disabled="true"                                                                                                                                                                          |
 | ⛔ **Table**                |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
-| ⛔ **Carousel**             |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| ✅ **Carousel**             |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Should this comment from above about `breakpoints: will be done after Grid Refactoring` be mentioned here again?

Reply:
Done